.lintstagedrc.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
   // as recommended by lint-staged.
   //
   // https://github.com/okonet/lint-staged#how-can-i-ignore-files-from-eslintignore
-  '*.{cjs,js,mjs}': filterTask('npm run lint:js:cli -- --fix'),
+  '*.{cjs,js,md,mjs}': filterTask('npm run lint:js:cli -- --fix'),
   '*.{json,md,yaml,yml}': 'npm run lint:prettier:cli -- --write',
   '*.scss': 'npm run lint:scss:cli -- --fix --allow-empty-input'
```

Comment:
**question**: Are we missing the `.md` extension here to automatically lint Markdown on commit?

Reply:
Yeah we were indeed. I've put an updated example in https://github.com/alphagov/govuk-frontend/pull/3698#discussion_r1213530634